tests/test_ast_symbol_extractor.py

```diff
@@ -298,6 +299,57 @@ def test_star_import():
     }
 
 
+def test_relative_import():
+    code = """
+    from . import core
+    from .core import ones
+    from ..core import twos
+    """
+    z = process_code_str(code)
+    assert z.symbols == {
+        "mm.core": {
+            "type": "relative-import",
```

Comment:
We don't really need a new type for relative imports, we could treat them as regular import with a level, added a type more for expressiveness- plus it's nicer to filter symbols by explicit type instead of having to check level to see if symbol is a relative import.